### ISSUE TITLE
Allow play/pause of physics

### ISSUE THREAD

**Issue by donmccurdy** [c-frame/aframe-extras#42]
At component or system level. Would allow dynamic bodies to be moved, use w/ editor, etc.

**Comment by Anjlo** [comment]
Thanks!

I was trying the body position set fix for a moveable/dragable entity but I think I'm implementing it wrong and can't get it working, so this would be a great addition.

Demo here: http://www.anj.fyi/drag (middle entities are dynamic-body)

**Comment by donmccurdy** [comment]
Sure thing! Fixed at b1fe200dca7cbeffa916ac45749dae5725e10b56 / v1.15.0

**Comment by donmccurdy** [comment]
Specifically, you can play/pause an element, or just a physics component:

``` javascript
el.pause(); // Stop all components

el.components['dynamic-body'].pause(); // Just stop physics
```
